src/odemis/driver/tescan.py

```diff
@@ -1127,12 +1161,22 @@ def _doMoveAbs(self, pos):
             logging.debug("Stage move to %s completed", pos)
             self._updatePosition()
 
-    def _doMoveRel(self, shift):
-        # Compute the position in absolute coordinates
+    def _doMoveRel(self, shift: dict):
+        """
+        Request the position in absolute coordinates and create new positions after applying shift values
+        :param shift (dict[str, float]): shift of linear axes in m or rotational axes in radians
+        """
         self._updatePosition()
         pos = self._position.copy()
+
+        # add the requested change to the current position
         for axis, change in shift.items():
             pos[axis] += change
+            if axis in ("rx", "rz"):
+                rng = self.axes[axis].range
+                if abs(rng[1] - rng[0]) > 2 * math.pi:
```

Comment:
I think you want to have a `>=`, to also include the (common) case where it's exactly reported as 360°.